packages/java-packages/snippet-replacer-maven-plugin/pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>azuresdk.plugin</groupId>
+  <artifactId>snippet-replacer-maven-plugin</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>maven-plugin</packaging>
+  <name>snippet-replacer-maven-plugin Maven Plugin</name>
+
+  <!-- FIXME change it to the project's website -->
+  <url>https://github.com/azure/azure-sdk-tools</url>
+
+  <prerequisites>
+    <maven>${maven.version}</maven>
+  </prerequisites>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <maven.version>3.3.9</maven.version>
```

Comment:
This is the version of the Maven plugins right and not the version of Maven this targets? If it is the latter, the Maven version it targets, you should use `3.6.x` as that is what pipelines is using.

Reply:
I'm not actually certain :P I'll update this.